### PR TITLE
Feature/52886 implement initial migration

### PR DESCRIPTION
**Pull request by AndreiMiron-hub**

### CODE REVIEW

File: source/Assist.Jully.2022.BE2/Assist.Jully.2022.BE2/Controllers/ListingController.cs [hunk 1]
```diff
@@ -5,7 +5,6 @@ namespace Assist.Jully._2022.BE2.Controllers
     [ApiController, Route("api/[controller]"), Produces("application/json")]
     public class ListingController : Controller
     {
-
         [HttpPost("Create")]
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.Jully.2022.BE2/Controllers/ListingController.cs:8]: "create"

File: source/Assist.Jully.2022.BE2/Assist.Jully.2022.BE2/Program.cs [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+using Assist.July._2022.BE2.Application.Interfaces;
+using Assist.July._2022.BE2.Application.Services;
+using Assist.July._2022.BE2.Domain.Entities;
+using Assist.July._2022.BE2.Infrastructure.Contexts;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.OpenApi.Models;
 var allowSpecificOrigins = "allowSpecificOrigins";
 var builder = WebApplication.CreateBuilder(args);
+var sqlConnectionBuilder = new SqlConnectionStringBuilder(builder.Configuration.GetConnectionString("SqlDBConnectionString"));
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.Jully.2022.BE2/Program.cs:10]: tabs , so you respect the 108 characters length convention

File: source/Assist.Jully.2022.BE2/Assist.Jully.2022.BE2/Program.cs [hunk 3]
```diff
@@ -34,6 +39,7 @@
     }
     );
 });
+builder.Services.AddDbContext<ApplicationDbContext>(options => options.UseSqlServer(sqlConnectionBuilder.ConnectionString));
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.Jully.2022.BE2/Program.cs:42]: tabs

File: source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Infrastructure/Contexts/ApplicationDbContext.cs [hunk 4]
```diff
@@ -1,29 +1,26 @@
...
     {
+        public ApplicationDbContext(DbContextOptions options) : base(options)
+        {
+        }
+        public DbSet<User> Users { get; set; }
+        public DbSet<Favorite> Favorites { get; set; }
+        public DbSet<Listing> Listings { get; set; }
+        public DbSet<Message> Messages { get; set; }
+        public DbSet<UserActivity> UserActivities { get; set; }
         protected override void OnModelCreating(ModelBuilder modelBuilder)
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Infrastructure/Contexts/ApplicationDbContext.cs:16]: new line

File: source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Infrastructure/Contexts/ApplicationDbContext.cs [hunk 5]
```diff
@@ -1,29 +1,26 @@
...
 using Assist.July._2022.BE2.Domain.Entities;
-
 namespace Assist.July._2022.BE2.Infrastructure.Contexts
 {
     public class ApplicationDbContext : DbContext
     {
+        public ApplicationDbContext(DbContextOptions options) : base(options)
+        {
+        }
+        public DbSet<User> Users { get; set; }
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Infrastructure/Contexts/ApplicationDbContext.cs:11]: new line

**Review by HincuAndreiAssist — APPROVED** [review]

**Pull request merged**